user-guide/README.rst

```diff
@@ -1,3 +1,20 @@
 Danube Cloud :: Documentation :: User Guide
 ###########################################
 
+Installation
+############
```

Comment:
Please use another character (`=`,`-`, etc.) for all subtitles (underline) in this file. The subtitles should be different from the main title.